certified-connectors/monday/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "1b5916f95df09bc7117981513b8ca99d",
```

Comment:
@ahlivehr secret look alike values are not allowed. Please replace it with a dummy value. you can use something like "Please enter you client Id"

Reply:
@Amjed-Ayoub Thanks for letting me know. I updated it to make it not look like a secret.